app-modules/ai/src/Services/TestAiService.php

```diff
@@ -96,4 +109,22 @@ public function getDeployment(): ?string
     {
         return null;
     }
+
+    public function supportsFileUploads(): bool
+    {
+        return true;
+    }
+
+    public function createFiles(AiMessage $message, array $files): Collection
+    {
+        return collect($files)->map(function ($file) {
```

Comment:
```suggestion
        return collect($files)->map(function ($file): AiMessageFile {
```

Can you add the correct param type too, is it SplFileObject or something else?

Reply:
Just a string since we aren't actually dealing with a file in the test service.